tests/integ/modin/index/test_index_name.py

```diff
@@ -0,0 +1,208 @@
+#
```

Comment:
You should add two separate tests here - one testing df.index.name renaming and if it affects regular index, and one to check whether renaming the regular index modifies df.index. If either/both don't work - test whether it is raising the correct error. Also add a test to check whether the warning you issue works.

Reply:
Will add a comment describing this